wolox-books/src/app/screens/auth/auth.component.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <!--   <head>
+    <meta charset="utf-8" />
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="application.css" />
+  </head> -->
+  <body>
```

Comment:
Since you already have a main `index.html`, you won't need this in here 😄

Reply:
Done 😄